build/crds-gen.sh

```diff
@@ -3,74 +3,70 @@
 . $(dirname "$0")/common.sh
 
 OPERATOR_SDK_VERSION=${1}
-ARC=$(echo $MACHTYPE | grep -oE '^[a-z0-9_]+')
-OS=$(echo $OSTYPE | sed 's/darwin/apple-darwin/g' | grep -oE '[a-z-]+')
+YQ_VERSION=${2}
+
+ARC=$(echo "${MACHTYPE}" | grep -oE '^[a-z0-9_]+')
+OS=$(echo "${OSTYPE}" | sed 's/darwin/apple-darwin/g' | grep -oE '[a-z-]+')
 OPERATOR_SDK_BUNDLE=operator-sdk-${OPERATOR_SDK_VERSION}-${ARC}-${OS}
 OPERATOR_SDK_URL=https://github.com/operator-framework/operator-sdk/releases/download/${OPERATOR_SDK_VERSION}/${OPERATOR_SDK_BUNDLE}
 
-unset GOPATH
 GO111MODULE=on
 # shellcheck disable=SC2155
 export GOROOT=$(go env GOROOT)
+OPERATOR_SDK_INSTALL_PATH=$(go env GOPATH)/bin
 
 installOperatorSDK() {
-  validateROOT
   printf "Installing Operator SDK version %s ..." "${OPERATOR_SDK_VERSION}"
   curl -LOs "${OPERATOR_SDK_URL}"
-  if grep -Fxq "Not Found" "${OPERATOR_SDK_BUNDLE}" &> /dev/null; then
+  if grep -Fxq "Not Found" "${OPERATOR_SDK_BUNDLE}" &>/dev/null; then
```

Comment:
Just curious, how does this work?

Reply:
This protects from incorrect Operator SDK installation. but we could use a `--fail` flag and remove this code.
Good catch, thanks, will fix it. Still working, but can be implement easily.